src/controllers/music/musicrecommended.js

```diff
@@ -1,49 +1,69 @@
-define(["browser", "layoutManager", "userSettings", "inputManager", "loading", "cardBuilder", "dom", "apphost", "imageLoader", "libraryMenu", "playbackManager", "mainTabsManager", "scrollStyles", "emby-itemscontainer", "emby-tabs", "emby-button", "flexStyles"], function(browser, layoutManager, userSettings, inputManager, loading, cardBuilder, dom, appHost, imageLoader, libraryMenu, playbackManager, mainTabsManager) {
+define(["browser", "layoutManager", "userSettings", "inputManager", "loading", "cardBuilder", "dom", "apphost", "imageLoader", "libraryMenu", "playbackManager", "mainTabsManager", "scrollStyles", "emby-itemscontainer", "emby-tabs", "emby-button", "flexStyles"], function (browser, layoutManager, userSettings, inputManager, loading, cardBuilder, dom, appHost, imageLoader, libraryMenu, playbackManager, mainTabsManager) {
     "use strict";
 
     function itemsPerRow() {
         var screenWidth = dom.getWindowSize().innerWidth;
-        return screenWidth >= 1920 ? 9 : screenWidth >= 1200 ? 12 : screenWidth >= 1e3 ? 10 : 8
+
+        if (screenWidth >= 1920) {
+            return 9;
+        }
+
+        if (screenWidth >= 1200) {
+            return 12;
+        }
+
+        if (screenWidth >= 1000) {
+            return 10;
+        }
+
+        return 8;
     }
 
     function enableScrollX() {
-        return !layoutManager.desktop
+        return !layoutManager.desktop;
     }
 
     function getSquareShape() {
-        return enableScrollX() ? "overflowSquare" : "square"
+        if (enableScrollX()) {
+            return "overflowSquare";
+        }
+
+        return "square";
```

Comment:
this doesn't seem needed IMHO

Reply:
https://github.com/jellyfin/jellyfin-web/pull/509/commits/47cc029c7056b2ad334a93ba80f1568c6020a0ad